### PR TITLE
Feature/dx 255 fixes round1

### PR DESCRIPTION
**Pull request by trieutran91**
Reorganized Invoice Seamless View to be more conform to Card Seamless View.
The biggest difference is the FinancingConsumer operation.

### CODE REVIEW

File: payments/invoice/seamless-view.md [hunk 1]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
 
-**POST Request**
+Different countries have different values for the properties. The table below
+showcase the values for the respective countries:
+
+**POST Request Options**
 
 {:.table .table-striped}
-| *POST Request* | **Sweden** ![Swedish flag][se-png] | **Norway** ![Norwegian flag][no-png] | **FInland** ![Finish flag][fi-png] |
+| *Country* | **Sweden** ![Swedish flag][se-png] | **Norway** ![Norwegian flag][no-png] | **FInland** ![Finish flag][fi-png] |
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]: When read this header does not make sense. This column is not countries.

File: payments/invoice/seamless-view.md [hunk 2]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
-    The logo will be displayed on all your invoices. Minimum accepted size
-    is 600x200 pixels, and at least 300 DPI.
-2. Included a link to "Terms and Conditions" for PayEx Faktura.
+Seamless View provides an integration of the payment process directly on your
+website. This solution offers a smooth shopping experience with Swedbank Pay
+payment pages seamlessly integrated in an `iframe` on your website. The costumer
+does not need to leave your webpage, since we are handling the payment in the
+`iframe` on your page.
+
+**TODO**
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]: Please resolve this TODO

File: payments/invoice/seamless-view.md [hunk 3]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
-    [`Authorize`][authorize] transaction, that is followed by a [`Capture`][capture] or [`Cancel`][cancel] request.
-* **Defining CallbackURL**: When implementing a scenario, it is optional
-    to set a [CallbackURL][callback-api]in the `POST` request.
-    If callbackURL is set PayEx will send a postback request to this URL when
-    the consumer has fulfilled the payment.
-    [See the Callback API description here.][callback-api]
+### Intent
 
-### Invoice flow
+{% include intent.md %}
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]: The include intent.md only has links for card.

File: payments/invoice/seamless-view.md [hunk 4]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
-    Merchant->>+PayEx: Post <invoice authorization> (Transaction Activity=FinancingConsumer)
-    PayEx-->>-Merchant: Transaction result
-    note left of Merchant: Fourth API request
-    Merchant->>+PayEx: GET <invoice payment>
-    PayEx-->>-Merchant: payment resource
-    Merchant-->>Consumer: Display result
-    deactivate Merchant
+### FinancingConsumer
+
+A `FinancingConsumer` payment is a straightforward way to an invoice for the
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]:
```suggestion
A `FinancingConsumer` payment is a straightforward way to invoice a
```

File: payments/invoice/seamless-view.md [hunk 5]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
+|  ✔︎︎︎︎︎  | `payment`                         | `object`      | The `payment` object contains information about the specific payment.                                                                                                                                                                                                                                                                               |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`operation`               | `string`      | The operation that the `payment` is supposed to perform. The [`FinancingConsumer`][financing-consumer] operation is used in our example. Take a look at the [create `payment` section][create-payment] for a full examples of the following `operation` options: [FinancingConsumer][financing-consumer], [Recur][recur], and [Verify][verify] |
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]: Why are only a few rows using the heavy check mark, while the rest of the tables does not?

File: payments/invoice/seamless-view.md [hunk 6]
```diff
@@ -23,107 +23,363 @@ sidebar:
...
+[authorize]: /payments/invoice/other-features#authorizations
+[callback]: /payments/invoice/other-features#callback
+[cancel]: /payments/invoice/after-payment#cancellations
+[capture]: /payments/invoice/after-payment#captures
+[create-payment]: /payments/invoice/other-features#create-payment
 [fi-png]: /assets/img/fi.png
+[financing-consumer]: /payments/invoice/other-features#financingconsumer
 [no-png]: /assets/img/no.png
+[payee-reference]: /payments/invoice/other-features/#payeereference
+[price-resource]: /payments/invoice/other-features/#prices
```
Comment (NullableInt) [inline payments/invoice/seamless-view.md]: These links will not work when deployed.

File: payments/invoice/seamless-view.md (hunk 3 above)
Comment (trieutran91) [reply payments/invoice/seamless-view.md]: Well spotted! The links are relative now

**Pull request merged**